packages/components-native/src/StatusIndicator/StatusIndicator.style.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from "react-native";
+import { tokens } from "../utils/design";
+
+const statusIndicatorDiameter = tokens["space-small"];
+
+export const styles = StyleSheet.create({
+  statusIndicator: {
+    borderRadius: tokens["radius-circle"],
+    backgroundColor: tokens["color-success"],
```

Comment:
I guess I could remove the default background color here since the color is being set with an inline style in the .tsx?